<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meCloth</class>
 <widget class="QWidget" name="meCloth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>283</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QFrame" name="frmClothPreset">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string>Preset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="preset">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnApplyPreset">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbOverwriteTextures">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string>Overwrite textures</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>215</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>35</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="ReTextureAvatar" name="warp_Kd">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="caption">
      <string>Warp diffuse</string>
     </property>
     <property name="ID">
      <string>warp_Kd</string>
     </property>
     <property name="breadcrumb">
      <string>Warp diffuse</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="ReTextureAvatar" name="warp_Ks">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="caption">
      <string>Warp specular</string>
     </property>
     <property name="ID">
      <string>warp_Ks</string>
     </property>
     <property name="breadcrumb">
      <string>Warp Specular</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" rowspan="2">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>164</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>164</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resources/RealityUI.qrc">:/images/WarpWeft.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="ReTextureAvatar" name="weft_Kd">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="caption">
      <string>Weft diffuse</string>
     </property>
     <property name="ID">
      <string>weft_Kd</string>
     </property>
     <property name="breadcrumb">
      <string>Weft diffuse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="ReTextureAvatar" name="weft_Ks">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="caption">
      <string>Weft specular</string>
     </property>
     <property name="ID">
      <string>weft_Ks</string>
     </property>
     <property name="breadcrumb">
      <string>Weft Specular</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="frURepeat">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Hor. repetition</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="uRepeat">
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>90000.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QFrame" name="frURepeat_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Ver. repetition</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="vRepeat">
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>90000.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>403</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>35</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReTextureAvatar</class>
   <extends>QWidget</extends>
   <header>ReTextureAvatar.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>uRepeat</tabstop>
  <tabstop>vRepeat</tabstop>
  <tabstop>preset</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/RealityUI.qrc"/>
 </resources>
 <connections/>
</ui>
